docs/pages/installation/minikube.md

```diff
@@ -59,16 +59,26 @@ Perform the following steps in order:
 
 ### 2. Install Meshery
 
-Follow the [installation steps](/guides/mesheryctl) to setup the mesheryctl CLI and install Meshery.
+Follow the [installation steps](/installation/quick-start) to setup the mesheryctl CLI and install Meshery.
 
-**Note**: If you are using docker driver, after completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
+**Users using docker driver**: 
+
+After completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
```

Comment:
```suggestion
After completing the Meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes cluster:
```
@leecalcote right?

Reply:
Almost, but not quite, no. The prepositional phrase "execute below command" is grammatically incorrect. There are number of other ways to phrase this, here's the suggestion that I made above: "...after completing the Meshery installation, execute the following command to..."

Reply:
```suggestion
After completing the Meshery installation, execute the following command to establish connectivity between Meshery and Kubernetes cluster:
```